README.md

```diff
@@ -81,6 +82,7 @@ In general, every development board has the following basic features:
 - ### Protection against [Reverse Voltage](https://www.google.com/url?sa=t&rct=j&q=&esrc=s&source=web&cd=&cad=rja&uact=8&ved=2ahUKEwjc8aaX1c3rAhXXXSsKHXphBgQQFjABegQICxAD&url=https%3A%2F%2Fwww.ti.com%2Flit%2Fpdf%2Fslva139&usg=AOvVaw0Qbub75JJ986MzLv6FYWKE) and Over Current
     - The SRA Board 2019 used diodes for reverse voltage protection in the power-line.
     - For the overcurrent protection of MCU and motor driver circuit, fuses of 300mA and 3A were used respectively.
+    - The SRA Board 2022 uses PTC Resettable fuses for overcurrent protection.
```

Comment:
Mention model number and configuration of PTC fuses
RUEF160 - I_hold = 1.6A, I_trip = 3.2A
RXEF050 - I_hold = 0.5 A, I_trip = 1A